tests/admin_views/admin.py

```diff
@@ -841,13 +841,21 @@ def view_on_site(self, obj):
 
 class WorkerInlineAdmin(admin.TabularInline):
     model = Worker
+    extra = 0
 
     def view_on_site(self, obj):
         return '/worker_inline/%s/%s/' % (obj.surname, obj.name)
 
 
+class ExtraWorkerInlineAdmin(WorkerInlineAdmin):
+    extra = 1
+
+    def get_formset_initial_data(self, request):
```

Comment:
It would be nice to add a test case where `extra != len(get_formset_initial_data)`.